protocol/x/perpetuals/client/cli/query_perpetual_test.go

```diff
@@ -66,6 +66,7 @@ func networkWithLiquidityTierAndPerpetualObjects(
 				Id:            uint32(i),
 				Ticker:        fmt.Sprintf("test_query_ticker_%d", i),
 				LiquidityTier: uint32(i % m),
+				MarketType:    types.PerpetualMarketType_PERPETUAL_MARKET_TYPE_CROSS,
```

Comment:
nit: Similar to the create n perpetuals, have this change between cross and isolated. Makes it so the test below will be testing querying isolated perpetuals.